lib/BC/SleighLifter.cpp

```diff
@@ -28,6 +28,20 @@
 
 namespace remill {
 
+namespace {
+
+static llvm::Value *ExtractOverflowBitFromCallToIntrinsic(
+    llvm::Intrinsic::IndependentIntrinsics intrinsic, llvm::Value *lhs,
+    llvm::Value *rhs, llvm::IRBuilder<> &bldr) {
+  llvm::Type *overloaded_types[1] = {lhs->getType()};
+  llvm::Function *target_instrinsic = llvm::Intrinsic::getDeclaration(
+      bldr.GetInsertBlock()->getModule(), intrinsic, overloaded_types);
+  std::array<llvm::Value *, 2> intrinsic_args = {lhs, rhs};
+  llvm::Value *res_val = bldr.CreateCall(target_instrinsic, intrinsic_args);
```

Comment:
I am pretty sure you can just do `bld.CreateCall(target_intrinsic, { lhs, rhs }` - it even more underlines the fact that they should have been a collection in the arguments as well.